changelog/Microsoft.DirectoryServices.json

```
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "13a51673-7133-48f3-bdfe-e6a8309de690",
+          "ApiChange": "Property",
+          "ChangedApiName": "allowedCloudEndpoints",
+          "ChangeType": "Addition",
+          "Description": "Added the **allowedCloudEndpoints** property to the [crossTenantAccessPolicy](https://docs.microsoft.com/en-us/graph/api/resources/crosstenantaccesspolicy?view=graph-rest-beta) resource.",
```

Suggestion to enhance the description.
```suggestion
          "Description": "Added the **allowedCloudEndpoints** property to the [crossTenantAccessPolicy](https://docs.microsoft.com/en-us/graph/api/resources/crosstenantaccesspolicy?view=graph-rest-beta) resource. Use this property to enable B2B collaboration across Azure clouds. For example, between tenants in Azure Global and Azure Government clouds, and Azure Global and Azure China clouds. ",
```